### ISSUE TITLE
Import default payloads of POST METHOD 

### ISSUE THREAD

**Issue by Sandeep06Dev** [rangav/thunder-client-support#706]
I am using swagger.json to get the list of web API , via IMPORT FROM URL but I noticed that the default payload of **POST** method is not showing in **BODY** tab. 

e.g. 
I was expecting this :
```
{
  "userName": "",
  "password": "",
  "error": "",
  "action": ""
}
```

but getting blank in json content
![image](https://user-images.githubusercontent.com/65675288/183697400-58a7559b-0c19-4d49-9413-76cb35c7a08c.png)

Thanks.

**Comment by rangav** [comment]
@Sandeep06Dev thanks for reporting the issue. 

Is it possible to share that swagger file, so I can test find the issue.

**Comment by Sandeep06Dev** [comment]
Sure,
[swagger (1).txt](https://github.com/rangav/thunder-client-support/files/9297190/swagger.1.txt)
Please change file extension to **json** eg: swagger.1.json
 Then Login / Sigin

**Comment by rangav** [comment]
Thanks @Sandeep06Dev for sharing the file, I have fixed the bug, will be part of next update. might release later today.

**Comment by Sandeep06Dev** [comment]
Ok. Waiting for that 😊

**Comment by Sandeep06Dev** [comment]
Hey @rangav  , Any plan for the next release?

**Comment by rangav** [comment]
Sorry missed yesterday, Will do the release today in next 2 hours.

**Comment by rangav** [comment]
I can upload the pre-release version of `vsix` here, you can use it in the meanwhile. let me know?

**Comment by Sandeep06Dev** [comment]
Yes. Please share vsix, i will install that manually.

**Comment by rangav** [comment]
ok sure, will upload in 5 - 10 minutes

**Comment by rangav** [comment]
Here is the vsix file. let me know if any feedback. see release notes for all features.

[vscode-thunder-client-1.17.3.vsix.zip](https://github.com/rangav/thunder-client-support/files/9308941/vscode-thunder-client-1.17.3.vsix.zip)

**Comment by KSandeep06** [comment]
> Here is the vsix file. let me know if any feedback. see release notes for all features in this update.
> 
> [vscode-thunder-client-1.17.3.vsix.zip](https://github.com/rangav/thunder-client-support/files/9308941/vscode-thunder-client-1.17.3.vsix.zip)

Yes working, Thanks .

**Comment by rangav** [comment]
This bug is now fixed and published to marketplace, please update to `v1.18.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v1.18.0

Please let me know your feedback.
